utils/zerovec/src/map/kv.rs

```diff
@@ -19,6 +19,8 @@ pub trait ZeroMapKV<'a> {
     /// The container that can be used with this type: [`ZeroVec`] or [`VarZeroVec`].
     type Container: MutableZeroVecLike<'a, Self, GetType = Self::GetType, OwnedType = Self::OwnedType>
         + Sized;
+    // TODO: Consider adding Slice here
+    // type Slice: ZeroVecLike<Self, GetType = Self::GetType> + ?Sized;
```

Comment:
If I do this, I could replace `<<K as ZeroMapKV<'a>>::Container as ZeroVecLike<K>>::BorrowedVariant` with `K::Slice`. Thoughts?